### PR TITLE
Fixed broken links-3.0

### PR DESCRIPTION
**Pull request by serenashe**
Signed-off-by: adakisme <serenashe@yunify.com>

### CODE REVIEW

**Comment by Sherlock113** [comment]
I notice some links have the same alias as the correct file name. Please double check that.

**Comment by serenashe** [comment]
> name

Modified. Please take a look.

File: content/en/docs/project-user-guide/image-builder/source-to-image.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ title: "Source to Image: Publish an App without a Dockerfile"
 keywords: 'KubeSphere, Kubernetes, Docker, S2I, Source-to-Image'
 description: 'Use S2I to import a Java sample project in KubeSphere, create an image and publish it to Kubernetes.'
 linkTitle: "Source to Image: Publish an App without a Dockerfile"
+aliases: 
+    - "/docs/project-user-guide/application-workloads/s2i-template/"
```
Comment (Sherlock113) [inline content/en/docs/project-user-guide/image-builder/source-to-image.md:7]: Delete this for now. This guide will be added later by @Felixnoo

File: layouts/robots.txt [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
+Disallow: /docs/project-user-guide/application-workloads/composing-app/
+Disallow: /docs/project-user-guide/application-workloads/ingress/
+Disallow: /docs/project-user-guide/storage/volume/
+Disallow: /docs/project-administration/project-quota/
+Disallow: /docs/project-administration/project-roles/
+Disallow: /docs/project-administration/project-members/
+Disallow: /docs/cluster-administration/storageclass/
+Disallow: /docs/devops-user-guide/introduction/pipeline/
+Disallow: /docs/devops-user-guide/introduction/credential/
+Disallow: /docs/project-user-guide/application-workloads/s2i-template/
```
Comment (Sherlock113) [inline layouts/robots.txt:14]: Delete the last string.

File: content/en/docs/project-user-guide/image-builder/source-to-image.md (hunk 1 above)
Comment (serenashe) [reply content/en/docs/project-user-guide/image-builder/source-to-image.md]: Deleted.

File: layouts/robots.txt (hunk 2 above)
Comment (serenashe) [reply layouts/robots.txt]: Done.

**Pull request merged**